src/apps/chifra/internal/blocks/handle_show.go

```diff
@@ -61,5 +61,8 @@ func (opts *BlocksOptions) HandleShowBlocks() error {
 		OutputFn:   opts.Globals.OutputFn,
 		Append:     opts.Globals.Append,
 		JsonIndent: "  ",
+		ModelExtraOptions: map[string]interface{}{
```

Comment:
Can we just call this 'Extra'? We're going to rewrite the way this works, but it just looks out of place to me. Sorry to be anal.